File: app/src/main/java/net/radityalabs/contactapp/presentation/util/BuildUtil.java
package net.radityalabs.contactapp.presentation.util;

import android.os.Build;

/**
 * Created by radityagumay on 4/17/17.
 */

public class BuildUtil {

    public static boolean isAndroidM() {
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.M;
    }

    public static boolean isAndroidKitkat() {
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT;
    }
}

File: app/src/main/java/net/radityalabs/contactapp/presentation/util/BuildUtil.kt
package net.radityalabs.contactapp.presentation.util

import android.os.Build

/**
 * Created by radityagumay on 4/17/17.
 */

object BuildUtil {

    val isAndroidM: Boolean
        get() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.M

    val isAndroidKitkat: Boolean
        get() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT
}
